SQL/MySQL.sublime-syntax

```diff
@@ -941,6 +977,11 @@ contexts:
 
   expressions:
     - meta_prepend: true
+    - match: \b(?i:declare)\b
+      scope: keyword.declaration.mysql
+      push:
+        - expect-type
+        - single-identifier
```

Comment:
What's the reason declare not being a `statement`? Is it realy valid within `( ... )` groups etc.?

Reply:
well spotted, thanks, this was a mistake :)